svgcheck/run.py

```diff
@@ -106,8 +112,13 @@ def main():
     if options.grey_scale:
         wp.color_threshold = options.grey_level
 
+    tempfile_in_use = False
     if len(args) < 1:
-        source = os.getcwd() + "/stdin"
+        with tempfile.NamedTemporaryFile(delete=False, mode="w+t") as tmp_file:
+            data = sys.stdin.read()
```

Comment:
This will bomb on files with unexpected encodings if that's a concern. You might be better off with
```python
with tempfile.NamedTemporaryFile(delete=False, mode="w+b") as tmp_file:
    data = sys.stdin.buffer.read()
```
which will let the parser cope with encoding issues.

Reply:
Addressed with 8aed510c73f1ce6ec4172a0d7ca2915797f95e60